website/docs/r/traffic_manager_profile.html.markdown

```diff
@@ -74,6 +74,8 @@ The following arguments are supported:
 
 * `monitor_config` - (Required) This block specifies the Endpoint monitoring configuration for the Profile, it supports the fields documented below.
 
+* `max_return` - (Optional) How many endpoints to return for DNS queries to this profile. Necessary when the `traffic_routing_method` is `MultiValue`, and ignored otherwise. Must be an integer between 1 and 8, with the default being 8.
```

Comment:
Can we adjust the wording a bit to be consistent?

```suggestion
* `max_return` - (Optional) The amount of endpoints to return for DNS queries to this Profile. Possible values range from `1` to `8`.

~> **NOTE**: `max_return` must be set when the `traffic_routing_method` is `MultiValue`.
```
**BTW**, there is no default being set?